stable/rabbitmq/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 4.11.1
+version: 4.12.1
```

The current version is `5.1.0` so you need to update to `5.2.0`
```suggestion
version: 5.2.0
```